src/Dedge.Cardizer/Dedge.Cardizer.fs

```diff
@@ -279,4 +279,25 @@ type Cardizer =
     /// </example>
     static member NextUatp () =
         Cardizer.GenerateCard [1] 15
-        
\ No newline at end of file
+
+    /// <summary>Returns a random RuPay number.</summary>
+    /// <returns>Random RuPay number</returns>
+    /// <example>
+    /// This sample shows how to call the <see cref="NextRuPay"/> method.
+    /// <code>
+    /// void PrintRuPay()
+    /// {
+    ///    Console.WriteLine(Cardizer.NextRuPay());
+    /// }
+    /// </code>
+    /// </example>
+    static member NextRuPay() =
```

Comment:
What about 3538 and 3561 (RuPay-JCB co-branded)? Do you plan to support them too?

Reply:
As discussed, we will add a new parameter to manage Cobranded cards